pkgs/development/interpreters/python/pypy/default.nix

```diff
@@ -195,10 +199,11 @@ in with passthru; stdenv.mkDerivation rec {
   enableParallelBuilding = true;  # almost no parallelization without STM
 
   meta = with lib; {
-    homepage = "http://pypy.org/";
+    homepage = "https://www.pypy.org/";
     description = "Fast, compliant alternative implementation of the Python language (${pythonVersion})";
     license = licenses.mit;
     platforms = [ "aarch64-linux" "x86_64-linux" "aarch64-darwin" "x86_64-darwin" ];
+    broken = optimizationLevel == 0; # generates invalid code
```

Comment:
If `optimizationLevel` is `"0"`, which seems supported based on the above test for membership in `[ "0" "1" "2" "3"]`, this test should presumably catch it, but `"0" != 0`.